src/popover/Popover.js

```diff
@@ -11,7 +11,7 @@ export const Props = {
   popover: t.struct({
     content: t.ReactChildren,
     attachToBody: t.maybe(t.Boolean),
-    position: t.maybe(t.enums.of(['top', 'bottom', 'left', 'right'])),
+    position: t.maybe(t.enums.of(['top', 'bottom', 'left', 'right', 'auto'])),
```

Comment:
As you're trying to compute both `position` and `anchor` I would add a new prop `auto: t.maybe(t.Boolean)` that should exist only if no `position` and `anchor` are passed